keycloak/sms-provider/src/main/java/org/sunbird/sms/SmsConfigurationType.java

```diff
@@ -0,0 +1,23 @@
+package org.sunbird.sms;
```

Comment:
Since we're storing the config in the json files, do we really need this file ? Since you are creating the http client in the provider itself.

Reply:
@coolbung Yes. Its like superset of parameters which the developer will look at when trying to fetch the configurations from the configs passed to configure() method of the specific provider.